### ISSUE TITLE
positive antimicrosomal antibodies

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#7602]
**Preferred term label:**
Anti-microsomal antibody positivity

**Synonyms**
Positive antimicrosomal antibodies

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies (immunoglobulins) in the blood circulation that reacts against microsomes.
PMID:28769306

**Parent term (use hpo.jax.org/app)**
HP:0030057 'Autoimmune antibody positivity'

**Comment:**
This can be an indication of an autoimmune thyroid disease such as Hashimoto's disease, idiopathic hypothyroidism, and Grave's disease.

This is needed for the INCLUDE Data Coordinating Center data annotations.
cc @lopierra

**Comment by nicolevasilevsky** [comment]
note to self, see comment:
https://github.com/obophenotype/human-phenotype-ontology/pull/7647#issuecomment-1228878313

**Comment by lopierra** [comment]
Regarding @pnrobinson's comment in #7647:

According to the [Wikipedia page](https://en.wikipedia.org/wiki/Antithyroid_autoantibodies), anti-microsomal antibodies were renamed anti-TPO antibodies. Our dataset has 2 fields:

1. Antimicrosomal antibodies [which = anti-TPO, and yes it is referring to thyroid microsomes]
2. TPO [which = anti-microsomal] or thyroglobulin antibodies [which are 2 separate things]

The common parent in HP is Autoimmune antibody positivity (HP:0030057), but it would be great if there were an intermediate grouping class named something like "antithyroid autoantibodies", and then we could use that term for both of the above fields.

**Comment by pnrobinson** [comment]
@lopierra  I am not sure that antithyroid autoantibodies because if there are antibodies that can react against epitopes located both in the thyroid and other places, then some people will think they are antithyroid and others will not, and so I think this may be confusing. Is there some accepted classification that we could use? Otherwise I would suggest we do not alter this and we add Antimicrosomal antibodies as a synonym to anti-TPO

**Comment by lopierra** [comment]
Sorry I wasn't clear - I agree that antimicrosomal antibodies should be a synonym of anti-TPO. **In addition** I was wondering if there can be an intermediate grouping called Anti-thyroid antibody positivity (or similar) that is a child of Autoimmune antibody positivity (HP:0030057), and would include Anti-TPO/microsomal (HP:0025379), anti-TSH (HP:0034189), and anti-thyroglobulin (HP:0032069).

I have seen these antibodies referred to as [Thyroid antibodies](https://medlineplus.gov/lab-tests/thyroid-antibodies/), [Anti-thyroid antibodies](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5422478/), and [Anti-thyroid autoantibodies](https://en.wikipedia.org/wiki/Antithyroid_autoantibodies) - I don't know if there is an officially accepted term.

**Comment by pnrobinson** [comment]
I am looking for definitions of what antithyroid autoantibodies are. Here is a start:

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5422478/

**Comment by lopierra** [comment]
I think you are asking me to write a definition? (Sorry if I misunderstood.) 

Anti-thyroid autoantibody positivity: 

The presence of antibodies directed against one or more components of the thyroid, including thyroid peroxidase (TPO), thyroglobulin (Tg), or thyrotropin receptor (also known as thyroid-stimulating hormone, or TSH, receptor). 

Reference: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5422478/

Children:
* Anti-thyroid-stimulating hormone receptor antibody positivity (HP:0034189), which is synonymous with Anti-thyrotropin receptor antibody (HP:0034116) [see #7717]
* Anti-thyroid peroxidase antibody positivity (HP:0025379)
* Anti-thyroglobulin antibody positivity (HP:0032069)

**Comment by pnrobinson** [comment]
@lopierra thanks for the suggestions, which I have implemented
- merged synonymous terms
- created Anti-thyroid antibody positivity parent term and moved the three other terms to be children
- added antimicrosomal antibodies as a synonym of anti-TPO.
